### PR TITLE
Initialize tooltips when a new a row is added in a subform

### PR DESCRIPTION
**Pull request by okonomiyaki3000**
Pull Request for Issue # .

### Summary of Changes
Like #12993 but for tooltips.

### Testing Instructions
Tooltips should still work normally. 
Tooltips within a new row added in a subform should also work.

### Documentation Changes Required
No.

### CODE REVIEW

**Comment by Quy** [comment]
Can this be expanded to include popovers?

**Comment by okonomiyaki3000** [comment]
This was initially part of a big pr that took care of several of this kind of thing and I was asked to separate them. So, no, I would not like to include that here. I'll do a different PR for it. But, yes, the principle is exactly the same. Subforms fire an event called `"subform-row-add"` which anything that needs initialization should listen for.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 8ff675059b917f41824094ee5d8ebf8ac9bcdd57<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12996">issues.joomla.org/tracker/joomla-cms/12996</a>.</sub>

**Comment by ReLater** [comment]
3.8.8-dev
I have overwritten layouts/joomla/form/renderlabel.php to have tooltips instead of popovers for labels in a subform.

```
#JHtml::_('bootstrap.popover');
#$classes[] = 'hasPopover';

JHtml::_('bootstrap.tooltip');
$classes[] = 'hasTooltip';
```

The "problem" I have is that tooltips are initialized and displayed correctly in new rows **also without this patch**.

**Comment by okonomiyaki3000** [comment]
@ReLater Doesn't sound like much of a problem. Can you provide any more details? Where can I see your code?

**Comment by Quy** [comment]
I can confirm as ReLater. It is an issue with popover. I assume it was an issue with tooltip too when I tested this PR, but that was not the case.

**Comment by okonomiyaki3000** [comment]
@Quy What is an issue with popover? This PR has nothing to do with popover.

**Comment by Quy** [comment]
The subform uses popover and not tooltip. The issue is that popover is displayed as browser's tooltip. See above where ReLater had to change the code from popover to tooltip.

popover

**Comment by okonomiyaki3000** [comment]
@Quy This PR has nothing to do with popover. I'll fix that in a different PR. This one is about tooltips. Yes, in fact tooltips already work even without this PR. There is a reason for that. When creating a new row in a subform, a function called `fixScripts` gets called which fixes a few limited things like this. In fact, it's the wrong way of handling it because it can only fix the things that subform knows about. The event-based solution is open ended and allows any new type of extension field to initialize itself instead of relying on subform.

File: media/system/js/subform-repeatable-uncompressed.js [hunk 1]
```diff
@@ -239,11 +239,6 @@
 			$row.find('.field-media-wrapper').fieldMedia();
 		}
 
-		// bootstrap tooltips
-		if($.fn.tooltip){
-			$row.find('.hasTooltip').tooltip({html: true, container: "body"});
-		}
-
```
Comment (okonomiyaki3000) [inline media/system/js/subform-repeatable-uncompressed.js:246]: @Quy This right here is what I'm talking about.

**Comment by okonomiyaki3000** [comment]
Here is one for popovers #20222

**Comment by ReLater** [comment]
I have tested this item :white_check_mark: successfully on 8ff675059b917f41824094ee5d8ebf8ac9bcdd57<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12996">issues.joomla.org/tracker/joomla-cms/12996</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12996">issues.joomla.org/tracker/joomla-cms/12996</a>.</sub>

**Pull request merged**